SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -696,23 +702,25 @@ void USpatialSender::SendOnEntityCreationRPC(UObject* TargetObject, UFunction* F
 #endif // !UE_BUILD_SHIPPING
 }
 
-void USpatialSender::SendCrossServerRPC(UObject* TargetObject, UFunction* Function, const SpatialGDK::RPCPayload& Payload,
+void USpatialSender::SendCrossServerRPC(UObject* TargetObject, UFunction* Function, SpatialGDK::RPCPayload Payload,
 										USpatialActorChannel* Channel, const FUnrealObjectRef& TargetObjectRef)
 {
 	const FRPCInfo& RPCInfo = ClassInfoManager->GetRPCInfo(TargetObject, Function);
 
 	Worker_ComponentId ComponentId = SpatialConstants::SERVER_TO_SERVER_COMMAND_ENDPOINT_COMPONENT_ID;
 
-	Worker_EntityId EntityId = SpatialConstants::INVALID_ENTITY_ID;
-	Worker_CommandRequest CommandRequest = CreateRPCCommandRequest(TargetObject, Payload, ComponentId, RPCInfo.Index, EntityId);
-
 	TOptional<Trace_SpanId> SpanId;
 	if (EventTracer != nullptr)
 	{
+		EventTraceUniqueId TraceId = true ? EventTraceUniqueId::GenerateUnique() : EventTraceUniqueId{};
```

Comment:
Ahh we should add user event code here :p

But again I think we should always call EventTraceUniqueId::GenerateUnique().

Reply:
I mean we are always calling it atm but.... you know what I mean

Reply:
user event code? Is that for being able to linearly resolve in user code?

Reply:
Oh no, just for the fact that you can not cause a cross server RPC with user events. We should add that functionality.

Reply:
That should really be a different PR but... I think it's two lines?

Reply:
Made a jira ticket for this.

Reply:
If it's easy to do I can add it here - I guess it's worth me checking that we can linearly resolve cross-server events!